include/itkMeshProcrustesAlignFilter.h

```diff
@@ -79,7 +79,7 @@ class ITK_TEMPLATE_EXPORT MeshProcrustesAlignFilter : public ProcessObject
   typedef DataObject::Pointer                                                DataObjectPointer;
   typedef typename OutputMeshType::CoordRepType                              CoordRepType;
   typedef vnl_matrix<CoordRepType>                                           MatrixType;
-  typedef AffineTransform<CoordRepType, 3>                                   TransformType;
+  typedef AffineTransform<double, 3>                                   TransformType;
```

Comment:
Required to avoid a warning when wrapping against ITK wrapped for `float` image pixel types.

Reply:
I was just about to ask "why this change" 😄

This upsets the alignment on the right. Adding [.clang-format](https://github.com/InsightSoftwareConsortium/ITKModuleTemplate/blob/main/%7B%7Bcookiecutter.project_name%7D%7D/.clang-format) should help. Maybe applying some other updates from module template too? One way to do that is to "start a new module". Enter the same information from this module. Once the module is generated compare them by copying the new content here and looking at the diff using git. That can be done in a later PR.